packages/user-group-service/src/groups/resolver.ts

```diff
@@ -17,25 +17,20 @@ export const GroupResolver = {
     },
     group ( root: any, { cn }: GraphQLArgs, ctx: any ) {
       return Groups
-        .find({ ldapCommonName: cn })
+        .find({ cn: cn })
         .exec()
         .then( res => res[0] );
-    },
-    findGroups ( root: any, { selector, limit }: GraphQLArgs, ctx: any ) {
-      return Groups
-        .find( selector )
-        .limit( limit )
-        .exec();
     }
   },
   Group: {
     members ( root: any, GraphQLArgs: any, ctx: any ) {
-      return UserGroupAPIHelper.roverFetch( `/groups/${ ( root[0] || root ).ldapCommonName }` )
+      return UserGroupAPIHelper.roverFetch( `/groups/${ ( root[ 0 ] || root ).cn }` )
```

Comment:
The root is always a singular object in a nested query. And if the parent resolver returns an array, this resolver executes for every item in the array separately, so the root will almost never be an array.
```suggestion
      return UserGroupAPIHelper.roverFetch( `/groups/${ root.cn }` )
```